TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -76,9 +76,7 @@ services:
         condition: service_started
       opensearchtarget:
         condition: service_started
-      trafficcomparator:
-        condition: service_healthy
-    command: /bin/sh -c "/runJavaWithClasspath.sh org.opensearch.migrations.replay.TrafficReplayer https://opensearchtarget:9200 --auth-header-value Basic\\ YWRtaW46YWRtaW4= --insecure --kafka-traffic-brokers kafka:9092 --kafka-traffic-topic logging-traffic-topic --kafka-traffic-group-id default-logging-group | nc trafficcomparator 9220"
+    command: /bin/sh -c "/runJavaWithClasspath.sh org.opensearch.migrations.replay.TrafficReplayer https://opensearchtarget:9200 --auth-header-value Basic\\ YWRtaW46YWRtaW4= --insecure --kafka-traffic-brokers kafka:9092 --kafka-traffic-topic logging-traffic-topic --kafka-traffic-group-id default-logging-group"
```

Comment:
For our Fargate deployed infrastructure, we shouldn't be writing replay output to CloudWatch since there will be lots of PII within it.
See https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_awslogs.html to confirm that both STDERR and STDOUT are shuffled off when the awslogs driver is used